.jscsrc

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
@scottgonzalez correct me if i'm wrong but isn't this just the jQuery preset with the bracket and paren spacing rules changed?